GVRf/Framework/framework/src/main/java/org/gearvrf/GVRConfigurationManager.java

```diff
@@ -192,6 +192,17 @@ public boolean isHomeKeyPresent() {
         return false;
     }
 
+    public void configureRendering(){
+        NativeConfigurationManager.configureRendering(mActivity.get().getNative());
```

Comment:
The activity ptr is cast to a ptr type of configuration manager, @apjagdale was this intentional?